packages/grafana-ui/src/components/RefreshPicker/_RefreshPicker.scss

```diff
@@ -2,3 +2,7 @@
   position: relative;
   margin-left: 10px;
 }
+
```

Comment:
Probably just lets add comment that this is needed because the button group does not handle second button being wrapped in a div.